### DISCUSSION TITLE
MorphTo single type

### DISCUSSION THREAD

**Discussion by aqjw** [laravel/nova-issues#4506] [Ideas & Feature Requests]
Could you please hide this select field?
Because when there is only one type, there is no reason for this field.

```
MorphTo::make('User', 'tokenable')->types([
    \App\Nova\Resources\User::class,
])->readonly(),
```

<img width="1269" alt="Screenshot 2022-06-29 at 19 56 59" src="https://user-images.githubusercontent.com/13101908/176493975-83fe9200-00f8-42f5-b55d-7bad5f33e223.png">

**Comment by crynobone** [comment]
> Because when there is only one type, there is no reason for this field.

Based on that argument, there's also no reason to use `MorphTo` when it can be `BelongsTo`

**Reply by aqjw** [reply]
In my case, this is the basic functionality of Sanctum. `MorphTo - tokenable`.
It is not advisable to change it, because in the future a new entity may appear that will require tokens.
